loguru/_colorizer.py

```diff
@@ -10,8 +10,8 @@ class Style:
     UNDERLINE = 4
     BLINK = 5
     REVERSE = 7
-    STRIKE = 8
-    HIDE = 9
+    STRIKE = 9
```

Comment:
Can you please swap the order of `HIDE` and `STRIKE` so that the list stays sorted?

Reply:
Sure, I'll do that within a few days.